docs/rules/core/0233-http-body.md

```diff
@@ -0,0 +1,64 @@
+---
+rule:
+  aip: 233
+  name: [core, '0233', http-body]
+  summary: Batch Create methods should use `*` as the HTTP body.
+---
+
+# Batch Create methods: HTTP body 
+
+This rule enforces that all `BatchCreate` RPCs use `*` as the HTTP `body`, as 
+mandated in [AIP-233][].
+
+## Details
+
+This rule looks at any RPC methods matching beginning with `BatchCreate`, and
+complains if HTTP `body` field is anything other than `*`.
+
+## Examples
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+rpc BatchCreateBooks(BatchCreateBooksRequest) returns (BatchCreateBooksResponse) {
+  option (google.api.http) = {
+    post: "/v1/{parent=publishers/*}/books:batchCreate"
+    // Http body is missing.
```

Comment:
Can the presentation here be revised to match the AIP-133 rule docs?

Reply:
Done.